File: MiniDOMWebCtrls/DMD/Tickets/TicketsHandler.vb

Imports minidom
Imports minidom.Forms
Imports minidom.Databases
Imports minidom.WebSite

Imports minidom.Office
Imports minidom.Sistema
Imports minidom.Anagrafica
Imports minidom.Web

Namespace Forms

    Public Class TicketsHandler
        Inherits CBaseModuleHandler

        Public Sub New()
            MyBase.New(ModuleSupportFlags.SCreate Or ModuleSupportFlags.SDelete Or ModuleSupportFlags.SDuplicate Or ModuleSupportFlags.SEdit Or ModuleSupportFlags.SAnnotations)

        End Sub


    End Class


End Namespace

File: MiniDOMWebCtrls/DMD/Tickets/TicketsHandler.cs

namespace minidom.Forms
{
    public class TicketsHandler : CBaseModuleHandler
    {
        public TicketsHandler() : base(ModuleSupportFlags.SCreate | ModuleSupportFlags.SDelete | ModuleSupportFlags.SDuplicate | ModuleSupportFlags.SEdit | ModuleSupportFlags.SAnnotations)
        {
        }
    }
}